vpr/src/draw/draw.cpp

```diff
@@ -2193,18 +2193,18 @@ ezgl::rectangle draw_get_rr_chan_bbox(int inode) {
                     + draw_coords->get_tile_width();
             bottom = draw_coords->tile_y[rr_graph.node_ylow(rr_node)]
                      + draw_coords->get_tile_width()
-                     + (1. + device_ctx.rr_nodes[inode].ptc_num());
+                     + (1. + rr_graph.node_ptc_num(rr_node));
```

Comment:
Since this is in the CHANX branch, we should use the node_track_num() API. Try to avoid using ``node_ptc_num()`` unless we have to